service/controller/v14/adapter/lifecycle_hooks.go

```diff
@@ -1,32 +0,0 @@
-package adapter
-
-import "github.com/giantswarm/aws-operator/service/controller/v14/key"
-
-// The template related to this adapter can be found in the following import.
-//
-//     github.com/giantswarm/aws-operator/service/controller/v14/templates/cloudformation/guest/lifecycle_hooks.go
-//
-
-type lifecycleHooksAdapter struct {
-	Worker lifecycleHooksAdapterWorker
-}
-
-type lifecycleHooksAdapterWorker struct {
-	ASG           lifecycleHooksAdapterASG
-	LifecycleHook lifecycleHooksAdapterLifecycleHook
-}
-
-type lifecycleHooksAdapterASG struct {
-	Ref string
-}
-
-type lifecycleHooksAdapterLifecycleHook struct {
-	Name string
-}
-
-func (a *lifecycleHooksAdapter) Adapt(config Config) error {
-	a.Worker.ASG.Ref = key.WorkerASGRef
-	a.Worker.LifecycleHook.Name = key.NodeDrainerLifecycleHookName
-
-	return nil
-}
```

Comment:
This is just moved. There are many types and adding `guest` prefix to the type name made git confused.